fdbserver/BlobManifest.actor.cpp

```diff
@@ -284,6 +284,8 @@ class BlobManifestLoader : public ReferenceCounted<BlobManifestLoader> {
 						if (e.code() == error_code_restore_missing_data) {
 							dprint("missing data for key range {} \n", granuleRange.toString());
 							TraceEvent("BlobRestoreMissingData").detail("KeyRange", granuleRange.toString());
+						} else {
+							TraceEvent("BlobManifestError").detail("KeyRange", granuleRange.toString());
```

Comment:
Nit: Include .error(e) in this trace event

Reply:
fixed. thank you!